easypermissions/src/main/java/pub/devrel/easypermissions/RationaleDialogFragment.java

```diff
@@ -14,6 +14,7 @@
  * {@link DialogFragment} to display rationale for permission requests when the request comes from
  * a Fragment or Activity that can host a Fragment.
  */
+@Deprecated
```

Comment:
What's the reason for marking this internal class `@Deprecated`?  Just for our own knowledge?

Reply:
Nope, that was just for me to mark stuff that used the basic `Activity`. Gonna need to refactor a bit.